impeller/renderer/backend/gles/proc_table_gles.cc

```diff
@@ -308,6 +308,22 @@ void ProcTableGLES::PushDebugGroup(const std::string& label) const {
   if (debug_label_max_length_ <= 0) {
     return;
   }
+
+  // glDebugMessageControl sometimes must be called before glPushDebugGroup:
+  // https://github.com/flutter/flutter/issues/135715#issuecomment-1740153506
+  //
+  // It's also not clear why it's necessary to call before each glPushDebugGroup
+  // call (one theory is separately for each EGL context), but it's necessary
+  // to get this to work on some modern Android devices.
+
+  // Setup of a default active debug group: Filter everything in.
+  DebugMessageControlKHR(GL_DONT_CARE,  // source
```

Comment:
Can this call be made less frequently?

I confirmed that on the Pixel6a/Mali driver `glDebugMessageControl` must be called once on a given EGL context before `glPushDebugGroup` calls are made on that context.  After that it does not need to be called again.

Not sure what the best place would be to set this up before the Impeller GLES backend calls `PushDebugGroup`.  One option would be doing this each time a `CommandBufferGLES` is constructed if `IMPELLER_DEBUG` is enabled.

Reply:
Possibly in reactor_gles https://github.com/flutter/engine/blob/main/impeller/renderer/backend/gles/reactor_gles.cc before executing any operations?

Reply:
I've added it in flush, and tested locally it appears to WAI.

Reply:
Done.